blockdata/processing/number.lisp

```diff
@@ -39,7 +39,7 @@
                  (begin
                    (if-not-zero   (is-first-block-in-conflation)
                                   (begin (vanishes!  (curr-NUMBER-hi))
-                                         (vanishes!  (curr-NUMBER-lo))))
+                                         (eq!        (curr-NUMBER-lo)    FIRST_BLOCK_NUMBER)))
```

Comment:
Fix of issue that was revealed by using the prover checker. Certain constraints only became active once traces included some zero padding. This was so far the only instance.